Feature/archive service

Anonycoders opened the pull request.

Anonycoders committed:
feature/archive-service
- service archive command created.

Anonycoders committed:
feature/archive-service
- delete_service_archive api method created.
- list_archived_services api method created.
- deploy_archived_service api method created.
- _check_for_manifest_errors private method created.
- archived param added to service_destroy command.
- service_dump description created.
- archive command group created.
- archive_list command created.
- archive_deploy command created.
- archive_delete command created.

Anonycoders committed:
feature/archive-service
- archives changed to service-archives.
- archive_deploy command removed.
- apply manifest divided into methods.
- from archive deployment moved to apply.

s-soroosh commented on fandogh_cli/service_commands.py:
```diff
@@ -181,99 +184,30 @@ def service_details(service_name):
-                        break
+@click.option('--from-archive', 'from_archive', is_flag=True, default=False,
+              help='deploy archived service')
+@click.option('--service', '-s', 'service_name', help='archive service name')
```
it's not needed

s-soroosh commented on fandogh_cli/service_commands.py:
```diff
@@ -181,99 +184,30 @@ def service_details(service_name):
-                        sys.exit(304)
-                    else:
-                        break
+@click.option('--from-archive', 'from_archive', is_flag=True, default=False,
```
doesn't need to be flag, can get the service name directly

Anonycoders committed:
feature/archive-service
- service archive urls edited.
- service_name flag removed.

Anonycoders committed:
feature/archive-service
- memory added to archive list table.

Anonycoders committed:
feature/archive-service
- --name and -n param added to archive delete command.
- archive_id replaced with archive_name.

Anonycoders committed:
feature/archive-service
- file prompt moved to service apply command body.

Anonycoders committed:
feature/archive-service
- service archive delete command prompt message changed to Service Archive Name.

Anonycoders committed:
feature/archive-service
- service archive delete confirm message changed.

Anonycoders committed:
feature/archive-service
- service archive delete command prompt message changed to Service Archive Name.
